ubuntu/gpu/cuda-11/rapids/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM databricksruntime/gpu-conda:cuda11
+
+COPY rapids-spec.txt /tmp/spec.txt
+
+RUN conda create --name rapids --file /tmp/spec.txt && \
+    rm -f /tmp/spec.txt
+
+# Set an environment variable used by Databricks to decide which conda environment to activate by default.
+ENV DEFAULT_DATABRICKS_ROOT_CONDA_ENV=rapids
```

Comment:
Does this work with newer runtimes without conda?

Reply:
Tried  to start the DCS cluster with DBR 9.1 and 10.4, both works. Added into the description.